doc/tutorials/transformer/te_init.ipynb

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
<p>In another ticket, we probably should create a function for that in the transformer extensions and also create a version in the notebook connector which directly uses the secret store.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>